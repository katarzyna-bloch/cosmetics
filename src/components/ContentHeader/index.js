import styled from 'styled-components'

const ContentHeader = styled.h5`
  font-size: 22px;
  text-align: center;
  padding-bottom: 10px;
  margin: 0;
`

export default ContentHeader
