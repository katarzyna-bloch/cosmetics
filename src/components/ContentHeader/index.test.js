import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ContentHeader from '.'

it('should render correctly', () => {
  let wrapped = shallow(<ContentHeader />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
