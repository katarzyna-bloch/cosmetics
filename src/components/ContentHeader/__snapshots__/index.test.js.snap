// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<h5
  className="sc-bdvvaa cRcTHD"
/>
`;
